fuzz/Cargo.toml

```diff
@@ -13,6 +13,7 @@ honggfuzz_fuzz = ["honggfuzz"]
 
 [dependencies]
 honggfuzz = { version = "0.5", optional = true }
+# honggfuzz = { git = "https://github.com/sanket1729/honggfuzz-rs", optional = true, rev="f357a544809e82bc04a08ef44e761fdca3b72de7" }
```

Comment:
Did you mean to add this line?

Reply:
Nope. Sorry, I was testing other things and this got through.